### ISSUE TITLE
❓ Summarization example : Why no shuffling ?

### ISSUE THREAD

**Issue by Colanim** [huggingface/transformers#3892]
# ❓ Questions & Help

Usually, when loading using Dataloader, data is shuffled for the training set. But in the case of the summarization example, data is not shuffled for training set :

https://github.com/huggingface/transformers/blob/1dc9b3c7847269961458c059ad8ad443b26bf60d/examples/summarization/bart/finetune.py#L105-L108

---

**Why data is not shuffled ?**

**Comment by sshleifer** [comment]
No good reason.
Do you feel comfortable sending a PR that shuffles for train loader only?
